src/domain/users-ips/index.ts

```diff
@@ -0,0 +1,17 @@
+export const parseIps = (
+  headers: null | NodeJS.Dict<string | string[]>,
+): IpAddress | null => {
+  if (!headers) return null
```

Comment:
Using `undefined` now to represent absence

Reply:
Setting as `null` here also breaks the checks at these points: https://github.com/GaloyMoney/galoy/search?q=%22ip+%3D%3D%3D+undefined%22&type=code

Sidenote: are there any ideas for having those checks not break when testing locally?

Reply:
Changed to `undefined` to represent absence.